spec/models/manageiq/providers/embedded_terraform/automation_manager/job_spec.rb

```diff
@@ -90,7 +97,7 @@
         {
           :input_vars                  => input_vars,
           :input_vars_type_constraints => {"name" => terraform_template_payload.dig(:input_vars, 0)},
-          :credentials                 => [],
+          :credentials                 => anything, # Authentication.where(credentials),
```

Comment:
I think we should try to match the expected `credentials` here not anything and comment this out